src/api.js

```diff
@@ -0,0 +1,12 @@
+import axios from "axios";
+const api = axios.create({
+  baseURL: "https://nc-news-m2v4.onrender.com/api",
+});
+
+const getArticles = () => {
+  return api.get("/articles").then((response) => {
+    return response.data.articles;
+  });
+};
+
```

Comment:
You have seeded your database with your test data and this is what is being rendered. In your back end, please seed the database with the dev data